app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -130,95 +134,120 @@ class Delegate extends Component<Props> {
+                />
               </div>
+              <StakePoolInfo pool={stakePool} gettingPoolInfo={gettingPoolInfo} />
+              <div />
+            </li>
+          </ul>
+        </div>
 
-              <div className="add-stake-pool-wrapper" />
-              <div className="delegation-info-row">
-                <label className="fee-label">
-                  Fee<AdaIcon />
-                </label>
-                <div className="delegation-fee">{printAda(delegationFee)}</div>
-              </div>
-              <div className="validation-row">
-                <button
-                  className="button primary"
-                  disabled={
-                    !isShelleyCompatible ||
-                    validationError ||
-                    calculatingDelegationFee ||
-                    stakePool.poolHash === ''
-                  }
-                  onClick={delegationHandler}
-                  {...tooltip(
-                    'You are using Shelley incompatible wallet. To delegate your ADA, follow the instructions to convert you wallet.',
-                    !isShelleyCompatible
-                  )}
-                >
-                  Delegate
-                </button>
-                {[
-                  calculatingDelegationFee ? (
-                    <CalculatingFee />
-                  ) : (
-                    <DelegationValidation
-                      delegationValidationError={delegationValidationError}
-                      txSuccessTab={txSuccessTab}
-                    />
-                  ),
-                ]}
-              </div>
-              {shouldShowTransactionErrorModal && (
-                <TransactionErrorModal
-                  onRequestClose={closeTransactionErrorModal}
-                  errorMessage={getTranslation(
-                    transactionSubmissionError.code,
-                    transactionSubmissionError.params
-                  )}
-                  showHelp={errorHasHelp(transactionSubmissionError.code)}
-                />
-              )}
-              {shouldShowConfirmTransactionDialog && <ConfirmTransactionDialog isDelegation />}
-            </Fragment>
-          }
-        />
+        <div className="add-stake-pool-wrapper" />
+        <div className="delegation-info-row">
+          <label className="fee-label">
+            Fee<AdaIcon />
+          </label>
+          <div className="delegation-fee">{printAda(delegationFee)}</div>
+        </div>
+        <div className="validation-row">
+          <button
+            className="button primary"
+            disabled={
+              !isShelleyCompatible ||
+              validationError ||
+              calculatingDelegationFee ||
+              stakePool.poolHash === ''
+            }
+            onClick={delegationHandler}
+            {...tooltip(
+              'You are using Shelley incompatible wallet. To delegate your ADA, follow the instructions to convert you wallet.',
```

Comment:
typo "convert you wallet" -> "convert your wallet"

Reply:
Thnx fixed.